### PR TITLE
update: enable camera model to be specified in config

### PR DESCRIPTION
**Pull request by jmcc94**

### CODE REVIEW

File: choices.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ module.exports = {
...
+	CAMERA: [
+		{id: 'Auto', label: 'Auto Detect'},
+		{id: 'P100', label: 'P100'},
+		{id: 'PF120', label: 'PF120'},
+		{id: 'P200A2A3', label: 'P200 A2_A3'},
+		{id: 'P200A4A5', label: 'P200 A4_A5'},
+		{id: 'A200GEN1', label: 'A200 GEN1'},
+		{id: 'A200GEN2', label: 'A200 GEN2'},
+		{id: 'A300GEN1', label: 'A300 GEN1'},
+		{id: 'A200GEN2', label: 'A200 GEN2'},
```
Comment (krocheck) [inline choices.js:30]: This should be A300GEN2?

**Comment by krocheck** [comment]
An upgrade script will be required to populate 'Auto' as the default for the new setting or a default should be applied inline if the setting is `undefined`.

File: choices.js (hunk 1 above)
Comment (jmcc94) [reply choices.js:30]: Correct. Fixed

**Comment by jmcc94** [comment]
> An upgrade script will be required to populate 'Auto' as the default for the new setting or a default should be applied inline if the setting is `undefined`.

upgrade script added

**Review by jmcc94 — COMMENTED** [review]
update: Filter api calls/actions/feddback/variables based on model & FW

**Pull request merged**